File: Assignment03/Assignment3_2.c

// INPUT: 24

// OUTPUT: 2   4   6   8   12

#include<stdio.h>

void DisplayFactor(int iNo)
{
    if(iNo<=0)
    {
        return;
    }
    for(int i=1;i<=iNo/2;i++)
    {
        if(iNo%i==0)
        {
            printf("%d\t",i);
        }
    }
    
}

int main()
{
    int iValue=0;
    printf("Enter Number\n");
    scanf("%d",&iValue);

    DisplayFactor(iValue);
    return 0;
}

File: Python/Assignment3/Assignment3_2.py
def Maximum(Arr,length):
    max=Arr[0]
    
    for i in range(length):
        if(max<Arr[i]):
            max=Arr[i]

    return max



def main():
    iRet=0
    print("Number of Elements:-")
    length=int(input())

    Arr=list()

    print("Enter the elements: ")
    for i in range(length):
        value=int(input())
        Arr.append(value)
    
    iRet=Maximum(Arr,length)
    print("Output:-",iRet)

if __name__=="__main__":
    main()